packages/react-components/src/atoms/__tests__/Dropdown.test.tsx

```diff
@@ -99,55 +175,82 @@ it('shows the value in red when invalid', () => {
       customValidationMessage="Nope."
     />,
   );
-  expect(findParentWithStyle(getByText('Heathrow'), 'color')?.color).toBe(
-    ember.rgb,
-  );
+  expect(findParentWithStyle(getByText('LHR'), 'color')?.color).toBe(ember.rgb);
 });
 
-it('gets a green border when focused', () => {
-  const { getByText } = render(
-    <Dropdown options={[{ value: 'LHR', label: 'Heathrow' }]} value="LHR" />,
+it('shows an error message when required field not filled', () => {
+  const { getByText, getByRole } = render(
+    <Dropdown
+      options={[{ value: 'LHR', label: 'Heathrow' }]}
+      value=""
+      required
+    />,
   );
-  expect(
-    findParentWithStyle(
-      getByText('Heathrow'),
-      'borderColor',
-    )?.borderColor.replace(/ /g, ''),
-  ).not.toBe(fern.rgb.replace(/ /g, ''));
-
+  userEvent.click(getByText('Select'));
   userEvent.tab();
-  expect(
-    findParentWithStyle(
-      getByText('Heathrow'),
-      'borderColor',
-    )?.borderColor.replace(/ /g, ''),
-  ).toBe(fern.rgb.replace(/ /g, ''));
+
+  expect(getByText('Please fill out this field.')).toBeVisible();
+  expect(getByRole('textbox')).toBeRequired();
 });
 
-it('gets greyed out when disabled', () => {
-  const { getByRole, getByText, rerender } = render(
+it('shows an error message  when required field has invalid value', async () => {
+  const { getByText, getByRole } = render(
     <Dropdown
       options={[{ value: 'LHR', label: 'Heathrow' }]}
-      value="LHR"
-      enabled={false}
+      customValidationMessage={'Please select something'}
```

Comment:
customValidationMessage is for Server Side validation messages. PLease look at TextField.tsx to see how this should work